scripts/Muon/fft_view.py

```diff
@@ -53,14 +53,18 @@ def __init__(self, parent=None):
         table_utils.setRowName(self.FFTTable,5,"Use Raw data")
         self.Raw_box= table_utils.addCheckBoxToTable(self.FFTTable,True,5)
 
-        options=['x','y','z']
-        table_utils.setRowName(self.FFTTable,6,"Axis")
-        self.axis= table_utils.addComboToTable(self.FFTTable,6,options)
+        table_utils.setRowName(self.FFTTable,6,"First Good Data")
```

Comment:
Similarly, a space after the commas in the argument list would make this easier to parse